src/App.js

```diff
@@ -17,7 +18,8 @@ const App = () => {
       par: 54,
       holes: 18,
     },
-    courses: []
+    courses: [],
+    error: null
```

Comment:
It ended up being important that we set this to null instead of an empty string.